lxd-user/callhook/callhook.go

```diff
@@ -73,6 +73,15 @@ func HandleContainerHook(lxdPath string, projectName string, instanceRef string,
 	u := api.NewURL().Path("internal", "containers", instanceRef, "on"+hook)
 	u.WithQuery("target", target)
 
+	if hook == "starthost" {
+		lxcPID := os.Getenv("LXC_PID")
```

Comment:
Can we lose this validation and instead still call the starthost URL with a missing or blank lxc_pid query parameter, then let LXD validate that its missing, as the error will then be more accessible in the LXD logs rather than in the lxc debug log.

Reply:
Fixed, thanks!